src/App.tsx

```diff
@@ -1,24 +1,34 @@
 import React from "react";
-import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
-
-import NavigationBar from "./Components/Navbar";
-import MyCarousel from "./Pages/Home";
-import AboutUs from "./Pages/About";
-import Home from "./Pages/Home";
-import DonationPage from "./Pages/Donation";
+import { Routes, Route, Link, BrowserRouter } from "react-router-dom";
+import { NavigationBar } from "./Components/Navbar/index";
+import HomePage from "./Pages/Home";
```

Comment:
```suggestion
import { HomePage } from "./Pages/Home";
```

To be consistent with other pages, make this change